tests/tasks/manage_test_interface.yml

```diff
@@ -17,9 +17,16 @@
 
 # veth
 - name: Create veth interface {{ interface }}
-  command: ip link add {{ interface }} type veth peer name peer{{ interface }}
```

Comment:
Please amend the commit message to explain why this is needed instead of only explaining what is done.

Reply:
Commit message updated.